wpilibc/src/main/native/include/PIDController.h

```diff
@@ -56,6 +57,8 @@ class PIDController : public LiveWindowSendable, public PIDInterface {
   virtual void SetContinuous(bool continuous = true);
   virtual void SetInputRange(double minimumInput, double maximumInput);
   virtual void SetOutputRange(double minimumOutput, double maximumOutput);
+  void SetMaxErrorToIntegrate(double maximumErrorToIntegrate);
```

Comment:
The function name is still ambiguous. The most obvious meaning to me based on that name is that this function sets the max error the integrator holds. The intended meaning is that this function sets the range within which the integrator integrates error.

Perhaps `SetErrorRangeToIntegrate(double maximumErrorMagnitude)` if what an integrator range is in `SetIntegratorRange(double maximumErrorMagnitude)` isn't clear? I still think `maximumErrorMagnitude` (and `m_maximumErrorMagnitude`) conveys its intent better than `maximumErrorToIntegrate` by the same ambiguity argument.